src/plugins/regions.ts

```diff
@@ -274,6 +286,16 @@ class SingleRegion extends EventEmitter<RegionEvents> {
     this.emit('update-end')
   }
 
+  private onContentClick(event: MouseEvent) {
+    event.stopPropagation()
+    const contentContainer = event.target as HTMLDivElement
+    contentContainer.focus()
```

Comment:
Why is this necessary btw? Isn't focusing the default action on click on editable elements?

Reply:
True, but with the region container having an event listener for clicks, this (focus on text content) does not happen. The event is immediately propagated to its parent. This function is in hopes that we can step in and handle the content property before propagating the event further. In the latest commit, I have made changes to emit the `region-clicked` event:
```js
private onContentClick(event: MouseEvent) {
    event.stopPropagation()
    const contentContainer = event.target as HTMLDivElement
    contentContainer.focus()
    this.emit('click', event)
  }
  ```